### PR TITLE
feat: alias -s to --show, update docs

### PR DESCRIPTION
**Pull request by caarlos0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,19 @@ Or, just install it with `go`:
...
 
+## Saving conversations
+
+All conversations are automatically saved, and can be identified by their latest
+prompt.
+
+A saved conversation has a SHA1 identifier and a title, similar to how Git
+works.
+
+Not so similar to Git, conversations can be updated, keeping their id but having
```
Comment (bashbunni) [inline README.md:103]: I would change this to "Unlike Git, conversations [...]"
```suggestion
Unlike Git, conversations can be updated, keeping their id but having
```

File: README.md [hunk 2]
```diff
@@ -234,18 +251,19 @@ Use the HTTP proxy to the connect the API endpoints.
...
 [MIT](https://github.com/charmbracelet/mods/raw/main/LICENSE)
 
-***
+---
 
 Part of [Charm](https://charm.sh).
 
 <a href="https://charm.sh/"><img alt="The Charm logo" width="400" src="https://stuff.charm.sh/charm-badge.jpg" /></a>
 
+<!--prettier-ignore-->
```
Comment (bashbunni) [inline README.md:268]: What does this do?

**Review by bashbunni — APPROVED** [review]
Looks great :)

File: README.md (hunk 2 above)
Comment (caarlos0) [reply README.md:268]: makes prettier not f up the line bellow :P

**Pull request merged**